FLT/Mathlib/LinearAlgebra/Determinant.lean

```diff
@@ -77,3 +78,29 @@ lemma IsSimpleRing.mulLeft_det_eq_mulRight_det' (d : Dˣ) :
     (LinearEquiv.mulLeft k d).det = (LinearEquiv.mulRight k d).det := by
   ext
   simp [mulLeft_det_eq_mulRight_det]
+
+
+/-!
+### Auxiliary lemmas about linear equivalences and matrices
+-/
+section LinearEquiv
+
+variable {F : Type*} [Field F]
+variable {ι : Type*} [Fintype ι] [DecidableEq ι]
+variable {V : Type*} [AddCommGroup V] [Module F V]
+
+lemma LinearEquiv.det_ne_zero
+  {F : Type*} [Field F] {V : Type*} [AddCommGroup V] [Module F V]
+  (e : V ≃ₗ[F] V) : e.toLinearMap.det ≠ 0 := isUnit_iff_ne_zero.mp (isUnit_det' e)
```

Comment:
```suggestion
lemma LinearEquiv.det_ne_zero
  {F : Type*} [CommRing F] [Nontrivial F] {V : Type*} [AddCommGroup V] [Module F V]
  (e : V ≃ₗ[F] V) : e.toLinearMap.det ≠ 0 := (isUnit_det' e).ne_zero
```

It's no harder to prove the result in the correct generality, so why not? `Nontrivial F` is just the assumption that `F` isn't the zero ring. If you can be bothered you could also change the F's to R's and the V's to M's ;-)